dodola/core.py

```diff
@@ -92,26 +92,27 @@ def build_xesmf_weights_file(x, method, target_resolution, filename=None):
     return str(out.filename)
 
 
-def xesmf_regrid(x, method, target_resolution, weights_path=None):
+def xesmf_regrid(x, domain, method, weights_path=None):
     """
 
     Parameters
     ----------
     x : xr.Dataset
     method : str
         Method of regridding. Passed to ``xesmf.Regridder``.
-    target_resolution : float
-        Decimal-degree resolution of global latxlon grid to regrid to.
     weights_path : str, optional
         Local path to netCDF file of pre-calculated XESMF regridding weights.
+    domain : xr.Dataset
```

Comment:
Suggest moving this doc for the `domain` parameter up under `x` and before `method` -- i.e. parameter order in the docstr needs to match the order in the function signature.

Reply:
Thanks for catching this!